pkg/micropython/doc.txt

```diff
@@ -0,0 +1,82 @@
+/**
+ * @defgroup pkg_micropython MicroPython RIOT port
+ * @ingroup  pkg
+ * @brief    MicroPython - Python for microcontrollers
+ *
+ * # MicroPython RIOT package
+ *
+ * "MicroPython is a lean and efficient implementation of the Python 3
+ *  programming language that includes a small subset of the Python standard
+ *  library and is optimised to run on microcontrollers and in constrained
+ *  environments."
+ *
+ * @see https://micropython.org
+ *
+ * ## Status
+ *
+ * MicroPython on RIOT has to be considered experimental. While the basic
+ * interpreter works fairly well on native and Cortex-M, it has not seen much
+ * testing.
+ *
+ * ## Configuration options
+ *
+ * The following defines (set using "CFLAGS += -Dfoo=bar" in the application
+ * Makefile) can be used to configure MicroPython:
+ *
+ * MP_RIOT_HEAPSIZE: heap size for Micropython. Defaults to 16KiB.
+ *
+ * ## Implementation details
+ *
+ * The RIOT port of MicroPython currently resides in a fork at
+ * https://github.com/kaspar030/micropython (in branch add_riot_port). It is
```

Comment:
Assuming 314da4a0b44a0566efc981761c6ee996bc332055 is going to stay, this should be updated.

Reply:
yup